cli/cmd/stat.go

```diff
@@ -248,7 +250,8 @@ func printStatTable(stats map[string]*row, resourceType string, w *tabwriter.Wri
 		"RPS",
 		"LATENCY_P50",
 		"LATENCY_P95",
-		"LATENCY_P99\t", // trailing \t is required to format last column
+		"LATENCY_P99",
+		"SECURED\t", // trailing \t is required to format last column
```

Comment:
i just realized this could get weird if we're also wanting to call this `MESHED` ?